src/runtime/pkg/kata-monitor/monitor.go

```diff
@@ -29,55 +31,48 @@ func SetLogger(logger *logrus.Entry) {
 
 // KataMonitor is monitor agent
 type KataMonitor struct {
-	sandboxCache         *sandboxCache
-	containerdAddr       string
-	containerdConfigFile string
-	containerdStatePath  string
+	runtimeEndpoint string
+	sandboxCache    *sandboxCache
 }
 
 // NewKataMonitor create and return a new KataMonitor instance
-func NewKataMonitor(containerdAddr, containerdConfigFile string) (*KataMonitor, error) {
-	if containerdAddr == "" {
-		return nil, fmt.Errorf("containerd serve address missing")
-	}
-
-	containerdConf := &srvconfig.Config{
-		State: defaults.DefaultStateDir,
+func NewKataMonitor(runtimeEndpoint string) (*KataMonitor, error) {
+	if runtimeEndpoint == "" {
+		return nil, fmt.Errorf("runtime endpoint missing")
```

Comment:
Nit: No formats so this could be `errors.New()`.

Reply:
done